test/manifests/kustomization.yaml

```diff
@@ -3,7 +3,7 @@ kind: Kustomization
 
 resources:
   - ../../config/cluster-install
-
+  - ../../config/extensions/webhook
```

Comment:
Is only adding this gonna make it work?

Reply:
Need to include this as well, adding in next commit:
```
  - patch: |-
      - op: add
        path: /spec/template/spec/containers/0/imagePullPolicy
        value: IfNotPresent
    target:
      kind: Deployment
      name: numaflow-webhook
```